src/time/format/formatter.cr

```diff
@@ -79,6 +83,12 @@ struct Time::Format
       io << get_short_day_name.upcase
     end
 
+    def short_day_name_with_comma?
```

Comment:
This is a duplicate definition.

Reply:
Where's the duplicate?

Reply:
Sorry I meant https://github.com/crystal-lang/crystal/pull/5123/files#diff-61fcbccde80bde3628b08077c5d0e22cR171 is overridden by https://github.com/crystal-lang/crystal/pull/5123/files#diff-819208e2dbfe2071e14f02023b5148d0R66.